test/mpi/errors/coll/rerr.c

```diff
@@ -14,7 +14,8 @@ int verbose = 0;
 
 int main(int argc, char *argv[])
```

Comment:
[test: do not test error case of MPI_BYTE and MPI_MAX] I remember we had a discussion around this issue with MPI_CHAR. Did we already previously allow MPI_BYTE+MPI_MAX? If not, I would not allow new, non-standard functionality in this PR.

Reply:
Agree. This is an example of the kind of behavior altering changes that it would be nice to isolate in their own PRs so they can be evaluated separately from the revamp/refactoring.

Reply:
Can do.